CurrencyDotNet.RestClient/Models/Requests/GetTickerRequest.cs

```diff
@@ -6,9 +6,14 @@ namespace CurrencyDotNet.RestClient.Models.Requests
     /// <summary>
     /// 24 hour rolling window price change statistics. Careful when accessing this with no symbol.
     /// </summary>
-    public class GetTicker24hrRequest : IRequestModel
+    public class GetTickerRequest : IRequestModel
```

Comment:
Maybe make a new hotfix branch and fix the inheritance from the base class. Or you can fix it in a new feature

Reply:
Fixed.